.travis.yml

```diff
@@ -1,9 +1,8 @@
 dist: bionic
 language: python
 python:
-    - "3.6"
-    - "3.7"
     - "3.8"
+    - "3.10"
```

Comment:
The tests failed because the newer numpy is not supported in python 3.7. Do we want to still support this Python version?

Reply:
I believe there are no limits like that. Anyone can do it.

Reply:
Actually, travis is obsolete I believe. Only github actions are used.

Reply:
What's the plan with the travis? Is it needed for anything?

Reply:
AFAIK travis is obsolete and can be removed.